packages/tooltip/src/Tooltip.ts

```diff
@@ -199,7 +199,26 @@ export class Tooltip extends SpectrumElement {
         });
     };
 
-    private closeOverlay = async (): Promise<void> => {
+    private closeOverlay = async (
+        event?: PointerEvent | FocusEvent | Event
+    ): Promise<void> => {
+        if (
+            event &&
+            event.type === 'pointerleave' &&
+            (event as PointerEvent).relatedTarget === this
+        ) {
```

Comment:
Similar code-as-docs:

```suggestion
        const pointerLeavingThis = event?.type === 'pointerleave' && (event as PointerEvent).relatedTarget === this;
        
        if (pointerLeavingThis) {
```